great_expectations/analytics/client.py

```diff
@@ -23,6 +27,7 @@ def submit(event: Event) -> None:
         groups = {
             "data_context": event.data_context_id,
         }
+
         if event.organization_id:
```

Comment:
In your previous commit you had an `and not get_config().anonymize_events` here. My suggestion (not very clear) was to add it to base_event.py as something like: 
```
    @property
    def anonymize(self) -> UUID | None:
        return get_config().anonymize_events
```
and then this line would be `if event.organization_id and not event.anonymize:`

Do we still need this check?

Reply:
I moved it into the base event.